### PR TITLE
Add apipath arg and some tests

### PR DESCRIPTION
**Pull request by nathanhruby**
## Pull Request Checklist

**Is this in reference to an existing issue?** No.

#### General

- [x] Update Changelog following the conventions laid out on [Keep A Changelog](http://keepachangelog.com/)

- [x] Update README with any necessary configuration snippets

- [x] Binstubs are created if needed

- [x] RuboCop passes

- [x] Existing tests pass 

#### Purpose
This adds an `--apipath` argument to allow one to specify the path of the
transport API when it shares the same port as web services.  This path also
adds some tests since there were not any.

#### Known Compatibility Issues
None known

### CODE REVIEW

**Comment by nathanhruby** [comment]
Check failures here are because the ruby 1.9 builds failed due to dev dependency issues (webmock -> addressable -> public_suffix needs ruby ~> 2.0).  

Unsure what the correct fix is here, use an older webmock or make plugin require ruby 2.0?

**Comment by nathanhruby** [comment]
Opened github/markup#999 about the version issue.  

Note that since most sensu-plugins projects require this package the trickle down effect may break all 1.9.3 builds

**Comment by nathanhruby** [comment]
Made sensu-plugins/documentation#7 to fix the FAQ too.

File: sensu-plugins-graylog.gemspec [hunk 1]
```diff
@@ -30,13 +30,13 @@ Gem::Specification.new do |s|
...
   s.require_paths          = ['lib']
-  s.required_ruby_version  = '>= 1.9.3'
+  s.required_ruby_version  = '>= 2.0.0'
   # s.signing_key            = File.expand_path(pvt_key) if $PROGRAM_NAME =~ /gem\z/
   s.summary                = 'Sensu plugins for graylog'
   s.test_files             = s.files.grep(%r{^(test|spec|features)/})
   s.version                = SensuPluginsGraylog::Version::VER_STRING
 
-  s.add_runtime_dependency 'sensu-plugin',   '1.2.0'
+  s.add_runtime_dependency 'sensu-plugin',   '1.4.4'
```
Comment (eheydrick) [inline sensu-plugins-graylog.gemspec]: Let's set the sensu-plugin dependency to `'~> 1.2'` to be consistent with other plugins.

File: .travis.yml [hunk 2]
```diff
@@ -4,7 +4,6 @@ cache:
 install:
 - bundle install
 rvm:
-- 1.9.3
 - 2.0
 - 2.1
 - 2.2
```
Comment (eheydrick) [inline .travis.yml:9]: add 2.3.0

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -24,5 +24,5 @@
 
 #### Purpose
 
-#### Known Compatablity Issues
+#### Known Compatibility Issues
```
Comment (eheydrick) [inline .github/PULL_REQUEST_TEMPLATE.md:27]: nice catch.

**Comment by eheydrick** [comment]
Left a few comments for small improvements. Dig the tests. 👍

**Comment by nathanhruby** [comment]
Cool, thanks for review.  Comments addressed in a7b6d8f

**Comment by nathanhruby** [comment]
Any chance this can get merged soon?  I still need to fix the other scripts but would like to make sure the test changes, etc.. are in first and avoid having a huge patch.

**Comment by nathanhruby** [comment]
Thank you Eric!  I'll have one more round of changes later this week.

**Pull request merged**